src/Components/css/fonts.css

```diff
@@ -0,0 +1,12 @@
+@import "@fontsource/roboto/100.css";
+@import "@fontsource/roboto/300.css";
+@import "@fontsource/roboto/400.css";
+@import "@fontsource/roboto/500.css";
+@import "@fontsource/roboto/700.css";
+@import "@fontsource/roboto/900.css";
```

Comment:
You can probably import a google font with a variable typeface that would have all the different font weights :)